quickwit/quickwit-indexing/src/source/mod.rs

```diff
@@ -138,7 +138,19 @@ use crate::source::ingest_api_source::IngestApiSourceFactory;
 /// 5MB seems like a good one size fits all value.
 const BATCH_NUM_BYTES_LIMIT: u64 = ByteSize::mib(5).as_u64();
 
-const EMIT_BATCHES_TIMEOUT: Duration = Duration::from_millis(if cfg!(test) { 100 } else { 1_000 });
+static EMIT_BATCHES_TIMEOUT: Lazy<Duration> = Lazy::new(|| {
+    if cfg!(any(test, feature = "testsuite")) {
```

Comment:
so the underlying issue was `feature = "testsuite"` missing here, and the rest is just to warn about bad environments?

Reply:
exactly!

Reply:
being able to tweak the actor heartbeat is a pretty big footgun, so I think it doesn´t hurt to warn against very bad settings